.zuul.yaml

```diff
@@ -3,7 +3,67 @@
     name: packit-service/packit-service
     check:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
     gate:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
+
+- import_playbook: ./files/zuul/{{ item }}-secret.yaml
```

Comment:
.zuul.yaml is just a yaml file, and will not be process by ansible. import_playbook won't work there. Instead you should add the secrets in that file directly. For each repository, Zuul look at a file .zuul.yaml or a directory zuul.d/*.yaml to find the zuul config elements, other files won't be read.